src/Panel/tabbedPanel.scss

```diff
@@ -1,4 +1,6 @@
-@import '../utils/theme/variables.scss';
+$tabbed-panel-tab-color: #3f4e49 !default;       // $gunmetal
```

Comment:
couldn't find an answer to this but I assume sass-loader is spawning a node-sass compilation for each top level import of a `.scss` from js, so we should be ok